components/page.js

```diff
@@ -12,9 +18,57 @@ const Content = styled.div`
   padding: 60px;
 `;
 
-export default ({ hackathons, currentPath, children }) => (
-  <Container>
-    <Sidebar currentPath={currentPath} hackathons={hackathons} />
-    <Content>{children}</Content>
-  </Container>
-);
+const HackathonNavBar = styled.div`
+  display: flex;
+  padding-left: 60px;
+`;
+
+const NavItem = styled.a`
+  display: block;
+  cursor: pointer;
+  font-size: 24px;
+  margin-right: 40px;
+  text-decoration: none;
+  color: ${p => (p.selected ? COLOR.BLACK : COLOR.DARK_COPY)};
+  border-bottom: ${p => (p.selected ? '3px solid black' : 'none')};
+`;
+
+export default ({ hackathons, currentPath, children }) => {
+  const [currPath, setCurrPath] = useState('intro');
+  useEffect(() => {
+    setCurrPath(window.location.pathname.split('/')[2]);
+  }, []);
+
+  return (
+    <Container>
+      <Sidebar currentPath={currentPath} hackathons={hackathons} />
+      <div>
+        <HomeHeader>{currentPath} Home</HomeHeader>
+        <HackathonNavBar>
+          <NavItem
+            key={`${currentPath}intro`}
+            href={`/${currentPath}/intro`}
+            selected={currPath === 'intro'}
+          >
+            Intro
+          </NavItem>
+          <NavItem 
+            key={`${currentPath}faq`} 
+            href={`/${currentPath}/faq`}
+            selected={currPath === 'faq'}
+          >
+            FAQ
+          </NavItem>
+          <NavItem
+            key={`${currentPath}sponsors`}
+            href={`/${currentPath}/sponsors`}
+            selected={currPath === 'sponsors'}
+          >
```

Comment:
since javascript files with the word sponsors on them get blocked by most adblockers, we're going to call the sponsorship page "spocos"

```suggestion
          <NavItem
            key={`${currentPath}spocos`}
            href={`/${currentPath}/spocos`}
            selected={currPath === 'spocos'}
          >
```